packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -243,6 +244,7 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
     // mark as read after chat thread is found in current state
     const markedChatThreads: string[] = [];
     this.notifyStateChange((draft: GraphChatListClient) => {
+      draft.status = 'chats read';
```

Comment:
Why add this? Nothing is reacting to this status.

Reply:
yes, we actually need the status to be different because if it is the same, it will stay at "chats loaded" which causes the onload to fire. Having it be specific means this onload does not fire which is the intent.